otwcode/otwarchive
# AO3-6069 Include lint dependencies in main Gemfile

# Pull Request Checklist

  as the *first* thing in your pull request title (e.g. `AO3-1234 Fix thing`)
  until they are reviewed and merged before creating new pull requests.

## Issue

https://otwarchive.atlassian.net/browse/AO3-6069

## Purpose

Solves my pet peeve of having to reinstall linting dependencies every time I rebuild my development image 😉

Includes linting dependencies in the default Gemfile, removes Gemfile.linters and associated references.

## Testing Instructions

- Clean slate (either build docker image or start from a clean ruby environment without any gems installed)
- `bundle install` from the project root
- `rubocop config/application.rb` should be a valid command and return lint output for the file

This should be reviewed by someone familiar with the deployment process.  As noted in JIRA I'm unsure how adding a new group in the Gemfile will affect the deployment.  Technically, it's only erb_lint that needs to be in the new group as the others will work fine in the development group.

Related question for reviewers: Any particular reason to pin versions on these dev dependencies?  Should we just upgrade them all to the latest compatible versions?

## References

https://otwarchive.atlassian.net/browse/AO3-6048
#3895 & #3901
